ops/lib.rs

```diff
@@ -912,7 +912,20 @@ fn codegen_u8_slice(core: &TokenStream2, idx: usize) -> TokenStream2 {
       Ok(b) => {
         let store = b.data() as *mut u8;
         // SAFETY: rust guarantees that lifetime of slice is no longer than the call.
-        unsafe { ::std::slice::from_raw_parts_mut(store, b.byte_length()) }
+        let slice = unsafe { ::std::slice::from_raw_parts_mut(store, b.byte_length()) };
```

Comment:
This is already UB

Reply:
how is `new Uint8Array(sabu8.buffer)` different? Some sort of internal syncronization?

Reply:
> how is `new Uint8Array(sabu8.buffer)` different? Some sort of internal syncronization?

We can check the source, but it shouldn't be implemented in a fashion that evokes C++ UB. (Here the issue is that Rust UB rules are broken*)

* Really Rust aliasing rules are quite broken across Deno anyways though..

Reply:
I've reverted back to copying SAB in JS